perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/RegistrarManagerMethod.java

```diff
@@ -479,14 +500,33 @@ public ApplicationFormItem call(ApiCaller ac, Deserializer parms) throws PerunEx
 	 * @param state List<String> List of states: NEW, VERIFIED, APPROVED, REJECTED
 	 * @return List<Application> Found applications
 	 */
+	/*#
+	 * Gets all applications in a given state for a given Group in a given date period.
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param state List<String> List of states: NEW, VERIFIED, APPROVED, REJECTED
+	 * @param dateFrom String Earliest date for applications
```

Comment:
Maybe suggest format for date string like ``YYYY-MM-DD``. Same for the VO version of the method.

Reply:
Fixed.